mongo/integration/collection_test.go

```diff
@@ -1067,6 +1067,65 @@ func TestCollection(t *testing.T) {
 			assert.Equal(mt, expectedModel, actualModel, "expected model %v in BulkWriteException, got %v",
 				expectedModel, actualModel)
 		})
+		mt.Run("unordered writeError index", func(mt *mtest.T) {
+			cappedOpts := bson.D{{"capped", true}, {"size", 64 * 1024}}
```

Comment:
Add a comment explaining why a capped collection is needed.

Reply:
done